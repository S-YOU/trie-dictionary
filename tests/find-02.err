nodes: 2
nodes: 4
nodes: 6
nodes: 8
nodes: 10
nodes: 12
nodes: 14
nodes: 16
nodes: 0
