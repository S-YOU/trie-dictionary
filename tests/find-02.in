find aba
insert aaaa
find aba
insert aaab
find aba
insert aaba
find aba
insert aabb
find aba
insert abaa
find aba
insert abab
find aba
insert abba
find aba
insert abbb
find aba
clear
find aba
