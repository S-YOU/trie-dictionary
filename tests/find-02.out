NO
word number: 0
NO
word number: 1
NO
word number: 2
NO
word number: 3
NO
word number: 4
YES
word number: 5
YES
word number: 6
YES
word number: 7
YES
cleared
NO
